### PR TITLE
bpo-31567: fix more decorator markup in docs (GH-3959)

### PR DESCRIPTION
**Pull request by merwok**
@dmiyakawa I missed these instances!

https://bugs.python.org/issue31567

### CODE REVIEW

File: Doc/library/typing.rst [hunk 1]
```diff
@@ -897,7 +897,7 @@ The module defines the following classes, functions and decorators:
 
    See :pep:`484` for details and comparison with other typing semantics.
 
-.. decorator:: no_type_check(arg)
+.. decorator:: no_type_check
 
    Decorator to indicate that annotations are not type hints.
```
Comment (merwok) [inline Doc/library/typing.rst:903]: fixing the text after this

**Comment by merwok** [comment]
I may use this PR to also address feedback received on the ticket.

**Comment by ilevkivskyi** [comment]
@merwok Please mention me and/or Guido when making changes to typing-related files (this will simplify life since typing is still developed in a separate repo).

**Comment by merwok** [comment]
Sure!  Could you have your names in the code owners file?

**Comment by ilevkivskyi** [comment]
>  Could you have your names in the code owners file?

This is an interesting idea. I am not a core-dev but @gvanrossum maybe you can add yourself as a code owner for ``typing.py`` and ``test_typing.py``?

**Comment by gvanrossum** [comment]
I'm happy to be listed as owner for typing.py etc. but I don't know how this is administered.

**Comment by merwok** [comment]
This file maps repo file patterns to github usernames or team names:
https://github.com/python/cpython/blob/master/.github/CODEOWNERS

**Pull request merged**